jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/publisher/impl/AnnotationPublisher.java

```diff
@@ -184,6 +185,9 @@ public void publish() throws Exception {
         // populateExistingIssuesForType(sampleAnnotation);
         String indexName = ESUtils.buildIndexNameFromAnnotation(sampleAnnotation);
         String typeIssue = ESUtils.getIssueTypeFromAnnotation(sampleAnnotation);
+        String targetType= sampleAnnotation.get(TARGET_TYPE);
+        List<Map<String, String>> typeList = RDSDBManager.executeQuery("SELECT displayName FROM cf_Target WHERE targetName ='"+targetType+"'");
+        String targetTypeDisplayName=typeList.get(0).get("displayName");

```

Comment:
safe to add a null check here. rest looks good.

Reply:
Done